WooCommerce/Classes/POS/Presentation/SimpleProductsOnlyInformation.swift

```diff
@@ -0,0 +1,95 @@
+import SwiftUI
+
+struct SimpleProductsOnlyInformation: View {
+    @Binding var isPresented: Bool
+
+    var body: some View {
+        VStack(spacing: Constants.contentBlockSpacing) {
+            VStack(spacing: Constants.textSpacing) {
+                Text(Localization.modalTitle)
+                    .font(.posModalTitle)
+
+                Group {
+                    Text(Localization.simpleProductsOnlyIssueMessage)
+                    Text(Localization.simpleProductsOnlyFutureMessage)
+                }
+                .font(.posBody)
+
+                VStack(spacing: Constants.textSpacing) {
+                    Text(Localization.modalHint)
+                        .font(.posDetail)
+
+                    Button { } label: {
+                        Label(Localization.modalAction, systemImage: "plus")
+                            .font(.posDetail)
+                    }
+                }
+                .frame(maxWidth: .infinity)
+                .padding(.vertical, Constants.hintVerticalPadding)
+                .padding(.horizontal, Constants.hintHorizontalPadding)
+                .background(Color(.quaternarySystemFill))
+                .clipShape(RoundedRectangle(cornerRadius: Constants.hintBackgroundCornerRadius))
+            }
+            .multilineTextAlignment(.center)
+
+            Button(action: {
+                isPresented = false
+            }) {
+                Text("OK")
```

Comment:
Just to check, since we already have "OK" in Localizable file, we can leave it since SwiftUI will use the translations automatically? Or should we use Localization enum for this also?

Reply:
Nope, that won't work because we use the reverse-dns keys for localization, not the plain English. Good spot, thanks.

Fixed in 858cf7c